azerothcore/azerothcore-wotlk
# fix(Core/Spells): Do not allow the use of the master's call while the pet is controlled [port from TC]

The hunter can use the master's call even if his pet is under the
effects of polymorphs, fears, etc.
This RP prevents the use of the master's call while the pet is under the
effects of stun, fear, disorientation.




## Changes Proposed:
-  fix(Core/Spells): Do not allow the use of the master's call while the pet is controlled.


## Issues Addressed:
- Closes 



## SOURCE:



## Tests Performed:

- Compiles on windows perfectly.
- 


## How to Test the Changes:
Log in with a hunter and another character, for example, a mage to polymorph. Apply polymorphism or fear to the pet and you will see that the master's call can no longer be used if the pet is under such effects.

## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
